app/models/base.rb

```diff
@@ -1,4 +1,8 @@
 class Base
   include Virtus.model(nullify_blank: true)
   include ActiveModel::Model
+
+  def scope
```

Comment:
I'd rename this method to something other than **scope**, maybe **locale_scope**. 

Why? Because **scope** means something else. And **locale_scope** points to exactly what we're using it for here.

What do you think?